tests/embedding/test_app.py

```diff
@@ -47,6 +47,9 @@ def test_deprecation_warning(embedded_app):
     struct = embedded_app.Model.AddStaticStructuralAnalysis()
     with pytest.warns(UserWarning):
         struct.SystemID
+    harmonic_acoustic = embedded_app.Model.AddHarmonicAcousticAnalysis()
+    with pytest.warns(UserWarning):
```

Comment:
minor - you can get SystemID from a harmonic acoustic.  So you don't really need a static structural anymore

Reply:
Done